src/main/java/kata/makers/ShoppingCart.java

```diff
@@ -0,0 +1,28 @@
+package kata.makers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShoppingCart {
+
+  private List<Integer> prices = new ArrayList<>();
```

Comment:
I think we can use Double instead(or that's on me, my requirement didn't state very clear), so we can cater different types of numbers